### ISSUE TITLE
Users need to know what it’s going to cost before they start

### ISSUE THREAD

**Issue by petegale** [alphagov/verify-local-patterns#225]
**Impact:** Users may be unwilling to start an application if the don’t know what it will cost them.

**Evidence:** Several users mentioned that they wanted to find out what the permit would cost for their car before they started.

**Comment by petegale** [comment]
![screen shot 2017-03-13 at 10 24 55](https://cloud.githubusercontent.com/assets/2970019/23850456/76ca207c-07d7-11e7-8f81-b04df896a646.png)

**Comment by EUzkuraityte** [comment]
link to #429

**Comment by EUzkuraityte** [comment]
Reopening this issue in light of new findings
![screen shot 2017-06-16 at 13 32 32](https://user-images.githubusercontent.com/20285115/27226725-46bb60d8-5298-11e7-87de-dafe8f5184a6.png)

**Comment by lizziebruce** [comment]
@sanjaypoyzer what do you think about having an expandable section "Prices for 3 or more permits" after the first para in the Costs sub-section?

**Comment by EUzkuraityte** [comment]
Adding further evidence from Cambridgeshire
![screen shot 2017-06-19 at 10 28 59](https://user-images.githubusercontent.com/20285115/27278864-8e970cd8-54da-11e7-9ced-bb068b56be04.png)
![screen shot 2017-06-19 at 10 53 06](https://user-images.githubusercontent.com/20285115/27279759-8c58db38-54dd-11e7-8621-990b6a0f6a32.png)

**Comment by lizziebruce** [comment]
Changes have been made to the prototype since this testing was carried out. The only outstanding issues I can see are: 

- [ ] costs for additional permits i.e. more than 2 (this also came up elsewhere)
- [x] remove words "up to" when council gives unlimited permits

Plus possible localisation for Cambridgeshire if they accept only very specific docs. Copy is as is because it represents a range of ID evidence that can be provided by the user. In informal testing of a group people generally knew their NI number.

**Comment by lizziebruce** [comment]
Moving to Is this a real need? I know a couple of users asked about costs for over 2 permits, but is this a need or a nice to have??

**Comment by EUzkuraityte** [comment]
@lizziebruce I think this is a real need as the 1st person we observed during the last session in newcastle mentioned having 5 cars.. So whilst he may not need to apply for 5 permits on one go, he needs to know how much it's going to cost him to get a permit for a 5th car, 6th car etc. We don't necessarily have to resolve this in the prototype but we should at least link to a LA page (we don't need to build it) that explains the costs for such permits.

**Comment by lizziebruce** [comment]
@EUzkuraityte the LA page would be part of the PP user journey as they wouldn't have separate info about PPs on their website. So we could make it a loop page, like the DOB checker on CT. I'd suggest making 'cost of additional permits' the link text.

**Comment by suryanaga** [comment]
I'd say this is definitely a real need. If they have more than 2 cars, they need to know much it's going to cost. Anyway we can hide this info from users that don't need to see it, whilst showing it to users who do though, is a plus.

**Comment by suryanaga** [comment]
I don't think it needs to be a whole page, the expandable section sounds sensible.
